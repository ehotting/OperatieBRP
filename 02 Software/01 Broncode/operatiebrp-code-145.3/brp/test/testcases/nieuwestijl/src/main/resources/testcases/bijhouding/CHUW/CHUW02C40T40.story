Meta:
@status                 Klaar
@usecase                UCS-BY.HG

Narrative: R2477 Bij nietigverklaring moet datum einde gelijk zijn aan datum aanvang

Scenario:   Reeds nietig verklaard huwelijk wordt gecorrigeerd naar  Relatie.Reden einde 'S' waarbij Relatie.Datum einde ongelijk is aan Relatie.Datum aanvang
            LT: CHUW02C40T40

Given alle personen zijn verwijderd
Given enkel initiele vulling uit bestand /LO3PL-CHUW/CHUW02C40T40-001.xls

Given pas laatste relatie van soort 1 aan tussen persoon 672738569 en persoon 600719881 met relatie id 30010001 en betrokkenheid id 30010001

!-- Nietigverklaren van het huwelijk
When voer een bijhouding uit CHUW02C40T40a.xml namens partij 'Gemeente BRP 1'

Then heeft het antwoordbericht verwerking Geslaagd

Given de database is aangepast met: update kern.his_relatie
                                  set    id = 9999
				  where  id = (
					         select hr.id
					         from   kern.his_relatie hr
					         join   kern.relatie r
					         on     r.id       = hr.relatie
					         where  r.srt      = 1
					         and    hr.dataanv = 20160510
					         and    hr.dateinde = 20160510
					      )

!-- Correctie van het huwelijk
When voer een bijhouding uit CHUW02C40T40b.xml namens partij 'Gemeente BRP 1'

Then heeft het antwoordbericht verwerking Geslaagd
Then is het antwoordbericht gelijk aan /testcases/bijhouding/CHUW/expected/CHUW02C40T40.xml voor expressie //brp:bhg_hgpCorrigeerHuwelijkGeregistreerdPartnerschap_R
