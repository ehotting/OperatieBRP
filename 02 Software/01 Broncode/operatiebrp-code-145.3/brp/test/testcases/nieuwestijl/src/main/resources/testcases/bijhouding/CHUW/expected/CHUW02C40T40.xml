<brp:bhg_hgpCorrigeerHuwelijkGeregistreerdPartnerschap_R xmlns:brp="http://www.bzk.nl/brp/brp0200">
    <brp:stuurgegevens>
        <brp:zendendePartij>199901</brp:zendendePartij>
        <brp:zendendeSysteem>BRP</brp:zendendeSysteem>
        <brp:referentienummer>*</brp:referentienummer>
        <brp:crossReferentienummer>00000000-0000-0000-0042-200000000104</brp:crossReferentienummer>
        <brp:tijdstipVerzending>*</brp:tijdstipVerzending>
    </brp:stuurgegevens>
    <brp:resultaat>
        <brp:verwerking>Geslaagd</brp:verwerking>
        <brp:bijhouding>Verwerkt</brp:bijhouding>
        <brp:hoogsteMeldingsniveau>Waarschuwing</brp:hoogsteMeldingsniveau>
    </brp:resultaat>
    <brp:meldingen>
        <brp:melding brp:objecttype="Melding" brp:referentieID="comid.correctieHuwelijk.correctievervalRelatie.huwelijk">
            <brp:regelCode>R2465</brp:regelCode>
            <brp:soortNaam>Waarschuwing</brp:soortNaam>
            <brp:melding>*</brp:melding>
        </brp:melding>
    </brp:meldingen>
    <brp:correctieHuwelijk brp:objecttype="AdministratieveHandeling">*</brp:correctieHuwelijk>
</brp:bhg_hgpCorrigeerHuwelijkGeregistreerdPartnerschap_R>
